orca-keel/src/test/kotlin/com/netflix/spinnaker/orca/keel/ImportDeliveryConfigTaskTests.kt

```diff
@@ -407,21 +415,26 @@ internal class ImportDeliveryConfigTaskTests : JUnit5Minutests {
                 manifestLocation.manifest,
                 manifestLocation.ref
               )
-            } throws RetrofitError.httpError(
+            } returns manifest
+          }
+          with(keelService){
+            every {
+              publishDeliveryConfig(manifest)
+            } throws SpinnakerHttpException(RetrofitError.httpError(
               "http://keel",
               Response(
                 "http://keel", 400, "", emptyList(),
-                JacksonConverter(objectMapper).toBody(parsingError) as TypedInput
+                converter.toBody(parsingError) as TypedInput
               ),
-              null, null
-            )
+              converter, null
+            ))
           }
         }
 
         test("task fails and includes the error details returned by keel") {
           val result = execute(manifestLocation.toMap())
           expectThat(result.status).isEqualTo(ExecutionStatus.TERMINAL)
-          expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(parsingError)
+          expectThat(result.context["error"]).isA<SpringHttpError>()
```

Comment:
Same deal here with restoring the equality check
```
      ✗ is equal to SpringHttpError(error=Bad Request, status=400, message=Parsing error, timestamp=2024-02-15T16:53:45.817Z, details={message=Parsing error, path=[{type=SomeClass, field=someField}], pathExpression=.someField})
              found SpringHttpError(error=Bad Request, status=400, message=Parsing error, timestamp=2024-02-15T16:53:45.823999Z, details={message=Parsing error, path=[{type=SomeClass, field=someField}], pathExpression=.someField})
```

Reply:
I have moved this test scenario to java file which uses wiremock. This is covered in the commit : https://github.com/spinnaker/orca/commit/7a48b311ff9454124496432a02fe12e1db9aae8e